app/src/main/java/com/celzero/bravedns/util/Constants.kt

```diff
@@ -15,158 +15,182 @@ limitations under the License.
+        // app type unknown
         const val UNKNOWN_APP = "Unknown"
 
-        //No package applications
+        // No package applications
         const val NO_PACKAGE = "no_package"
 
         // Number of network log entries to store in the database.
         const val TOTAL_NETWORK_LOG_ENTRIES_THRESHOLD = 5000
 
+        // invalid application uid
         const val INVALID_UID = -1
+
+        // missing uid, used when the uid is undermined. see ConnectionTracer#getUidQ()
         const val MISSING_UID = -2000
 
+        // label for rethinkdns plus doh endpoint
         const val RETHINK_DNS_PLUS = "RethinkDNS Plus"
-        const val RETHINK_DNS = "RethinkDNS Basic (default)"
-
-        const val BRAVE_DNS_BASE_NAME = "bravedns"
-        const val RETHINK_DNS_BASE_NAME = "rethinkdns"
 
-        // maximum time delay before sending block connection response.
+        // maximum time delay before sending block connection response
         val DELAY_FIREWALL_RESPONSE_MS: Long = TimeUnit.SECONDS.toMillis(30)
 
-        const val REFRESH_APP_DURATION = 3L
+        // constants used as part of intent to load the viewpager's screen
+        const val VIEW_PAGER_SCREEN_TO_LOAD = "view_pager_screen"
 
-        const val SCREEN_TO_LOAD = "view_pager_screen"
+        // name-value to pass as part of intent
+        // determines whether launched from local/remote
+        const val BLOCKLIST_LOCATION_INTENT_EXTRA = "location"
 
-        const val LOCATION_INTENT_EXTRA = "location"
-        const val STAMP_INTENT_EXTRA = "stamp"
-        const val URL_INTENT_EXTRA = "url"
+        // stamp name-value for blocklist configure screen
+        const val BLOCKLIST_STAMP_INTENT_EXTRA = "stamp"
 
-        const val VPN_INTENT = "android.net.vpn.SETTINGS"
+        // url name-value for blocklist configure screen
+        const val BLOCKLIST_URL_INTENT_EXTRA = "url"
 
+        // default custom http proxy port number
         const val HTTP_PROXY_PORT = "8118"
 
+        // default custom socks5 ip
         const val SOCKS_DEFAULT_IP = "127.0.0.1"
+
+        // default custom socks5 port
         const val SOCKS_DEFAULT_PORT = "9050"
+
+        // constants to send type of proxy: for socks5
         const val SOCKS = "Socks5"
 
+        // data-time format used as part of network log adapter
         const val DATE_FORMAT_PATTERN = "HH:mm:ss"
 
+        // represents the orbot proxy mode (see AppMode#ProxyMode)
         const val ORBOT_PROXY = 10L
 
-        // Represents the download source of the application. playstore/fdroid/website
-        const val DOWNLOAD_SOURCE_PLAY_STORE = 1
-        const val DOWNLOAD_SOURCE_FDROID = 2
-        const val DOWNLOAD_SOURCE_WEBSITE = 3
-
-        // Constants generated as part of BuildConfig.FLAVORS
+        // constants generated as part of BuildConfig.FLAVORS (playstore/fdroid/website)
         const val FLAVOR_PLAY = "play"
         const val FLAVOR_FDROID = "fdroid"
         const val FLAVOR_WEBSITE = "website"
```

Comment:
move  these to strings.xml also for wherever used for ui

Reply:
Created string-literals in `strings.xml` for the UI usage.